images/node/Dockerfile

```diff
@@ -1,29 +1,17 @@
 #
-# This is an OpenShift node image with integrated OpenvSwitch SDN.
-#
-# This image expects to have a volume mounted at /etc/origin/node that contains
-# a KUBECONFIG file giving the node permission to talk to the master and a
-# node configuration file.
+# This is the OpenShift OpenvSwitch SDN image.
 #
 # The standard name for this image is openshift/origin-node
```

Comment:
If this comment is still true it seems wrong, and probably both the image and this directory should be renamed?

Reply:
Still debating how we're going to change this.  A new image is very expensive organizationally, so we're probably going to go stepwise here